test/util.cpp

```diff
@@ -32,6 +35,35 @@ std::string loadFile(const std::string& filename) {
     throw std::runtime_error("Error opening file");
 }
 
+bool operator==(const mapbox::geometry::feature<short>& a, const mapbox::geometry::feature<short>& b) {
+    // EXPECT_EQ(a.geometry, b.geometry);
+    EXPECT_EQ(a.properties, b.properties);
+    return true;
+}
+
+bool operator==(const mapbox::geometry::feature_collection<short>& a, const mapbox::geometry::feature_collection<short>& b) {
+    EXPECT_EQ(a.size(), b.size());
+    unsigned i = 0;
+    for (i = 0; i < a.size(); i++) {
+        EXPECT_EQ(a[i] == b[i], true);
+    }
+    return true;
+}
+
+bool operator==(const std::map<std::string, mapbox::geometry::feature_collection<short>>& a, const std::map<std::string, mapbox::geometry::feature_collection<short>>& b) {
+    EXPECT_EQ(a.size(), b.size());
+    // @TODO iterate through all tiles in a & confirm in b && vice versa
```

Comment:
I'll add this comparison in but at first glance it looks like the tiles generated are quite different in count + possibly tile coordinate from `master`. @mourner may ping you for guidance on whether this is an expected update to the algorithm in `geojson-vt-cpp` (and maybe also whether there's a JS test fixture reference/parity expected).